scripts/sort-makefile-am-sources.pl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+
+while (<>) {
+    # print "read1: $_";
+    if (! /^(nodist_)?tests_test.*_SOURCES/) {
+        print;
+        next;
+    }
+    print;
+    # accumulate files and prep for sorting
+    my %files;
+    while (<>) {
+        # print "read2: $_";
+        chop;
```

Comment:
IIRC, `chomp` is safer:

```suggestion
        chomp;
```

Reply:
should be roughly equivalent in this case, but sure